src/components/chat/w2wChat/profile/Profile.tsx

```diff
@@ -34,15 +36,15 @@ const Profile = ({ setActiveTab }: any): JSX.Element => {
         >
           <ImageV2
             alt="Profile"
-            src={connectedUser.profilePicture}
+            src={ensAvatar ?? connectedUser.profilePicture}
           />
         </ItemVV2>
         <SpanV2
           color={theme.default.secondaryColor}
           size="16px"
           weight="400"
         >
-          {caip10ToWallet(connectedUser.wallets).slice(0, 8) + '...' + caip10ToWallet(connectedUser.wallets).slice(-7)}
+          {ensName ?? caip10ToWallet(connectedUser.wallets).slice(0, 8) + '...' + caip10ToWallet(connectedUser.wallets).slice(-7)}
```

Comment:
@oscgu We are not able to see the ens name of the logged in user.

Reply:
Hi, where exactly is the name missing?

This is what I'm seeing on my end
![ensname](https://user-images.githubusercontent.com/94227101/203117013-cfe20bb4-4133-4acd-88db-dec5f3168400.png)

Reply:
I can't reproduce your issue, have you tried waiting a little bit longer because of the public rpc ratelimit?

Reply:
The wallet doesn't have a primary ens name set. You can check/set that here https://app.ens.domains/address/0xffa1af9e558b68bbc09ad74058331c100c135280

Reply:
okay got it let me check